psi4/src/psi4/libmints/oeprop.h

```diff
@@ -266,26 +430,18 @@ class OEProp : public Prop {
     /// Compute field at specified grid points
     void compute_field_over_grid();
 
+    MultipolePropCalc mpc;
```

Comment:
Protected variables must have a appended `_`, so `mpc_` etc.

Reply:
Thank you for noticing this, I will amend the pull request!